src/RecordTask.cc

```diff
@@ -1554,16 +1554,17 @@ pid_t RecordTask::find_newborn_thread() {
   }
 }
 
-pid_t RecordTask::find_newborn_child_process() {
+pid_t RecordTask::find_newborn_process(bool sibling) {
```

Comment:
I dislike boolean parameters, they're hard to read at call sites. Can you just pass the parent pid here instead?